dao/mongo.go

```diff
@@ -282,7 +283,12 @@ func (m *MongoService) GetPaymentsWithRefundPendingStatus() ([]models.PaymentRes
 	collection := m.db.Collection(m.CollectionName)
 	statusFilter := bson.M{refundStatus: "pending"}
 
-	paymentDBResources, err := collection.Find(context.Background(), statusFilter)
+	options := options.Find()
```

Comment:
Rename var to avoid clash with `options` import

Reply:
Fixed. Thanks